docs/src/SUMMARY.md

```diff
@@ -103,3 +103,4 @@
 - [ADR-015: ITF: informal trace format](./adr/015adr-trace.md)
 - [ADR-016: ReTLA: Relational TLA](./adr/016adr-retla.md)
 - [PDR-017: Checking temporal properties](./adr/017pdr-temporal.md)
+- [ADR-018: Inlining](./adr/018adr-inlining.md)
```

Comment:
It seems like this is more specific than just the problem of inlining in general?